tests/should_succeed/octalnuber.jou

```diff
@@ -0,0 +1,7 @@
+from "stdlib/io.jou" import printf
+
+def main()-> int:
+    printf("%d", 0o777) # Should be 511
+    printf("%d", 0o337522) # Should be 114514
+    printf("%d", 0o7244314) # Should be 1919180

```

Comment:
Two changes: you need to use `\n` to get these to print on separate lines, and you need to use `Output:` comments so that the test runner knows what to expect.
```suggestion
    printf("%d\n", 0o777) # Output: 511
    printf("%d\n", 0o337522) # Output: 114514
    printf("%d\n", 0o7244314) # Output: 1919180
```

Reply:
just found it at the same time :)